galaxy/templates/deployment-job.yaml

```diff
@@ -28,8 +28,8 @@ spec:
           image: alpine:3.7
           command: ['sh', '-c', 'until nc -z -w3 {{ template "galaxy-postgresql.fullname" $ }} 5432; do echo waiting for galaxy-postgres service; sleep 1; done;']
         - name: {{ .Chart.Name }}-init-mounts
-          image: alpine:3.7
-          command: ['sh', '-c', 'chown -R 101:101 /galaxy/server/database && install -o 101 -g 101 /galaxy/server/config/integrated_tool_panel.xml /galaxy/server/config/editable/integrated_tool_panel.xml']
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          command: ['sh', '-c', 'chown -R 101:101 /galaxy/server/database && install -o 101 -g 101 /galaxy/server/config/integrated_tool_panel.xml /galaxy/server/config/editable/integrated_tool_panel.xml; if [ ! -f "/galaxy/server/database/config/editable_shed_tool_conf.xml" ]; then mkdir -p /galaxy/server/database/config && cp /galaxy/server/config/shed_tool_conf.xml.sample /galaxy/server/database/config/editable_shed_tool_conf.xml; fi']
```

Comment:
Given the _editable_ `integrated_tool_panel.xml` is in the `config/editable` folder, shouldn't this follow suit?

Reply:
`config/editable` is not on the mounted persistent volume.
It's not a problem for the integrated tool panel because we just copy it from the values file and it doesn't need to be persisted, but this file needs to be copied only the first time when it doesn't exist, and stay on the volume when the pods die, so it has to be under `/database` or wherever the PV is mounted. On that note, actually, we don't set the path for the mounted volume in values, and we have it hardcoded for many purposes at this point, should we externalize it to values under `persistence.mountPath` or something similar and have it be imported from there where it's referenced in the manifests to make it easier to change in the future?